go/result_set.go

```diff
@@ -268,12 +302,43 @@ func (res ResultSet) GetErrorCode() ErrorCode {
 	return ErrorCode(int64(res.resp.ErrorCode))
 }
 
-func (res ResultSet) GetErrorMsg() []byte {
+func (res ResultSet) GetLatency() int32 {
+	return res.resp.LatencyInUs
+}
+
+func (res ResultSet) GetSpaceName() string {
+	if res.resp.Comment == nil {
+		return ""
+	}
+	return string(res.resp.SpaceName)
+}
+
+func (res ResultSet) GetErrorMsg() string {
 	if res.resp.ErrorMsg == nil {
-		var empty []byte
-		return empty
+		return ""
+	}
+	return string(res.resp.ErrorMsg)
+}
+
+func (res ResultSet) GetPlanDesc() *graph.PlanDescription {
+	if res.resp.Comment == nil {
+		return graph.NewPlanDescription()
+	}
+	return res.resp.PlanDesc
+}
+
+func (res ResultSet) GetComment() string {
+	if res.resp.Comment == nil {
+		return ""
 	}
-	return res.resp.ErrorMsg
+	return string(res.resp.Comment)
+}
+
+func (res ResultSet) IsEmpty() bool {
+	if res.resp.Data == nil || res.resp.Data.ColumnNames == nil || res.resp.Data.Rows == nil {
```

Comment:
if res.resp.Data is not null, res.resp.Data.ColumnNames and res.resp.Data.Rows is not null ,
`if res.resp.Data == nil || len(res.resp.Data.Rows) == 0`